hnn_core/hnn_io.py

```diff
@@ -15,6 +15,244 @@
 from .cell_response import CellResponse
 from .externals.mne import fill_doc
 
+def network_to_markdown(net, fname=None):
```

Comment:
If this is a `write_xxx` function, it should follow the API in the remaining codebase

Reply:
parameters should be documented in the docstring